### PR TITLE
N°6907 - Update chinese translations

### PR DESCRIPTION
**Pull request by bdejin**
1. Added the missing translation (ended with ~)
2. Updated the existing ones using below standard:
-  a space after . , ) ' " ? ]
-  a space before ( ' " [
-  no space before or after /

### CODE REVIEW

File: datamodels/2.x/combodo-db-tools/dictionaries/zh_cn.dict.combodo-db-tools.php [hunk 1]
```diff
@@ -27,48 +27,48 @@
...
-	'DBTools:Indication' => '重要: 修复数据库错误后,可能会出现新的不一致,您必须重新运行一次分析.',
-	'DBTools:Disclaimer' => '免责申明: 在应用修复之前,应先备份数据库',
+	'DBTools:Indication' => '重要: 修复数据库错误后, 可能会出现新的不一致, 您必须重新运行一次分析.',
+	'DBTools:Disclaimer' => '免责申明: 在应用修复之前, 应先备份数据库',
 	'DBTools:Error' => '错误',
 	'DBTools:Count' => '个数',
-	'DBTools:SQLquery' => 'SQL 查询',
-	'DBTools:FixitSQLquery' => 'SQL query To Fix it (indication)~~',
-	'DBTools:SQLresult' => 'SQL 结果',
+	'DBTools:SQLquery' => 'SQL语句',
```
Comment (chileeb) [inline datamodels/2.x/combodo-db-tools/dictionaries/zh_cn.dict.combodo-db-tools.php]: 这里保留使用“SQL查询”是不是更好一些？

File: datamodels/2.x/itop-config-mgmt/dictionaries/zh_cn.dict.itop-config-mgmt.php [hunk 2]
```diff
@@ -130,7 +130,7 @@
 	'Class:FunctionalCI/Attribute:finalclass' => '类型',
 	'Class:FunctionalCI/Attribute:finalclass+' => '根本属性的名称',
 	'Class:FunctionalCI/Tab:OpenedTickets' => '活跃的工单',
-	'Class:FunctionalCI/Tab:OpenedTickets+' => 'Active Tickets which are impacting this functional CI~~',
+	'Class:FunctionalCI/Tab:OpenedTickets+' => '影响当前功能性配置项的活跃工单',
```
Comment (chileeb) [inline datamodels/2.x/itop-config-mgmt/dictionaries/zh_cn.dict.itop-config-mgmt.php]: 功能配置项

File: datamodels/2.x/itop-incident-mgmt-itil/dictionaries/zh_cn.dict.itop-incident-mgmt-itil.php [hunk 3]
```diff
@@ -110,10 +110,10 @@
 	'Class:Incident/Attribute:urgency/Value:4+' => '低',
 	'Class:Incident/Attribute:origin' => '来源',
 	'Class:Incident/Attribute:origin+' => '',
-	'Class:Incident/Attribute:origin/Value:in_person' => 'In-person~~',
-	'Class:Incident/Attribute:origin/Value:in_person+' => 'Incident created following a face-to-face discussion~~',
-	'Class:Incident/Attribute:origin/Value:chat' => 'Chat~~',
-	'Class:Incident/Attribute:origin/Value:chat+' => 'Incident created following a ~~',
+	'Class:Incident/Attribute:origin/Value:in_person' => '当面',
+	'Class:Incident/Attribute:origin/Value:in_person+' => '创建于当面沟通后的事件',
+	'Class:Incident/Attribute:origin/Value:chat' => '聊天',
```
Comment (chileeb) [inline datamodels/2.x/itop-incident-mgmt-itil/dictionaries/zh_cn.dict.itop-incident-mgmt-itil.php]: 聊天工具

File: datamodels/2.x/itop-incident-mgmt-itil/dictionaries/zh_cn.dict.itop-incident-mgmt-itil.php [hunk 4]
```diff
@@ -110,10 +110,10 @@
...
+	'Class:Incident/Attribute:origin/Value:chat+' => '创建于聊天后的事件',
```
Comment (chileeb) [inline datamodels/2.x/itop-incident-mgmt-itil/dictionaries/zh_cn.dict.itop-incident-mgmt-itil.php]: 聊天工具的事件

File: datamodels/2.x/itop-portal-base/dictionaries/zh_cn.dict.itop-portal-base.php [hunk 5]
```diff
@@ -79,9 +79,9 @@
 	'Brick:Portal:UserProfile:Password:Title' => '密码',
 	'Brick:Portal:UserProfile:Password:ChoosePassword' => '新密码',
 	'Brick:Portal:UserProfile:Password:ConfirmPassword' => '确认密码',
-	'Brick:Portal:UserProfile:Password:CantChangeContactAdministrator' => '要修改密码,请联系管理员',
+	'Brick:Portal:UserProfile:Password:CantChangeContactAdministrator' => '要修改密码, 请联系管理员',
 	'Brick:Portal:UserProfile:Password:CantChangeForUnknownReason' => '无法修改密码, 请联系管理员',
-	'Brick:Portal:UserProfile:PersonalInformations:Title' => '个体信息',
+	'Brick:Portal:UserProfile:PersonalInformations:Title' => '个人信息',
```
Comment (chileeb) [inline datamodels/2.x/itop-portal-base/dictionaries/zh_cn.dict.itop-portal-base.php]: 人员信息

File: datamodels/2.x/itop-request-mgmt-itil/dictionaries/zh_cn.dict.itop-request-mgmt-itil.php [hunk 6]
```diff
@@ -110,10 +110,10 @@
...
 	'Class:UserRequest/Attribute:origin' => '来自',
 	'Class:UserRequest/Attribute:origin+' => '',
-	'Class:UserRequest/Attribute:origin/Value:in_person' => 'In-person~~',
-	'Class:UserRequest/Attribute:origin/Value:in_person+' => 'Request created following a face-to-face discussion~~',
-	'Class:UserRequest/Attribute:origin/Value:chat' => 'Chat~~',
-	'Class:UserRequest/Attribute:origin/Value:chat+' => 'Request created following a chat discussion~~',
+	'Class:UserRequest/Attribute:origin/Value:in_person' => '当面',
+	'Class:UserRequest/Attribute:origin/Value:in_person+' => '创建于当面讨论后的需求',
+	'Class:UserRequest/Attribute:origin/Value:chat' => '聊天',
+	'Class:UserRequest/Attribute:origin/Value:chat+' => '创建于聊天后的需求',
```
Comment (chileeb) [inline datamodels/2.x/itop-request-mgmt-itil/dictionaries/zh_cn.dict.itop-request-mgmt-itil.php]: 聊天工具的需求

File: datamodels/2.x/itop-request-mgmt-itil/dictionaries/zh_cn.dict.itop-request-mgmt-itil.php [hunk 7]
```diff
@@ -110,10 +110,10 @@
 	'Class:UserRequest/Attribute:urgency/Value:4+' => '低',
 	'Class:UserRequest/Attribute:origin' => '来自',
 	'Class:UserRequest/Attribute:origin+' => '',
-	'Class:UserRequest/Attribute:origin/Value:in_person' => 'In-person~~',
-	'Class:UserRequest/Attribute:origin/Value:in_person+' => 'Request created following a face-to-face discussion~~',
-	'Class:UserRequest/Attribute:origin/Value:chat' => 'Chat~~',
-	'Class:UserRequest/Attribute:origin/Value:chat+' => 'Request created following a chat discussion~~',
+	'Class:UserRequest/Attribute:origin/Value:in_person' => '当面',
+	'Class:UserRequest/Attribute:origin/Value:in_person+' => '创建于当面讨论后的需求',
+	'Class:UserRequest/Attribute:origin/Value:chat' => '聊天',
```
Comment (chileeb) [inline datamodels/2.x/itop-request-mgmt-itil/dictionaries/zh_cn.dict.itop-request-mgmt-itil.php]: 聊天工具

File: datamodels/2.x/itop-request-mgmt-itil/dictionaries/zh_cn.dict.itop-request-mgmt-itil.php [hunk 8]
```diff
@@ -248,12 +248,12 @@
...
 	'Class:UserRequest/Method:ResolveChildTickets' => '解决子工单',
-	'Class:UserRequest/Method:ResolveChildTickets+' => '递归解决子工单(自动解决),并调整相关字段与父级工单保持一致: 服务, 团队, 办理人, 解决方案信息',
+	'Class:UserRequest/Method:ResolveChildTickets+' => '递归解决子工单 (自动解决), 并调整相关字段与父级工单保持一致: 服务, 团队, 办理人, 解决方案信息',
 ));
 
 
 Dict::Add('ZH CN', 'Chinese', '简体中文', array(
 	'Organization:Overview:UserRequests' => '此组织的所有用户需求',
-	'Organization:Overview:MyUserRequests' => '我在此组织内发起的需求',
+	'Organization:Overview:MyUserRequests' => '我在此组织的用户需求',
```
Comment (chileeb) [inline datamodels/2.x/itop-request-mgmt-itil/dictionaries/zh_cn.dict.itop-request-mgmt-itil.php]: 建议保留原翻译

File: datamodels/2.x/itop-request-mgmt/dictionaries/zh_cn.dict.itop-request-mgmt.php [hunk 9]
```diff
@@ -116,10 +116,10 @@
...
 	'Class:UserRequest/Attribute:origin' => '来源',
 	'Class:UserRequest/Attribute:origin+' => '',
-	'Class:UserRequest/Attribute:origin/Value:in_person' => 'In-person~~',
-	'Class:UserRequest/Attribute:origin/Value:in_person+' => 'Request created following a face-to-face discussion~~',
-	'Class:UserRequest/Attribute:origin/Value:chat' => 'Chat~~',
-	'Class:UserRequest/Attribute:origin/Value:chat+' => 'Request created following a chat discussion~~',
+	'Class:UserRequest/Attribute:origin/Value:in_person' => '当面',
+	'Class:UserRequest/Attribute:origin/Value:in_person+' => '创建于当面讨论后的需求',
+	'Class:UserRequest/Attribute:origin/Value:chat' => '聊天',
+	'Class:UserRequest/Attribute:origin/Value:chat+' => '创建于聊天后的需求',
```
Comment (chileeb) [inline datamodels/2.x/itop-request-mgmt/dictionaries/zh_cn.dict.itop-request-mgmt.php]: 聊天工具的用户需求

File: datamodels/2.x/itop-structure/dictionaries/zh_cn.dict.itop-structure.php [hunk 10]
```diff
@@ -170,8 +170,8 @@
 	'Class:Person/Attribute:team_list+' => '这个人归属的所有团队',
 	'Class:Person/Attribute:tickets_list' => '工单',
 	'Class:Person/Attribute:tickets_list+' => '此人发起的所有工单',
-	'Class:Person/Attribute:user_list' => 'Users~~',
-	'Class:Person/Attribute:user_list+' => 'All the Users associated to this person~~',
+	'Class:Person/Attribute:user_list' => '用户',
+	'Class:Person/Attribute:user_list+' => '所有关联到此个人的用户',
```
Comment (chileeb) [inline datamodels/2.x/itop-structure/dictionaries/zh_cn.dict.itop-structure.php]: 此人员

File: datamodels/2.x/itop-structure/dictionaries/zh_cn.dict.itop-structure.php [hunk 11]
```diff
@@ -302,7 +302,7 @@
 //
 
 Dict::Add('ZH CN', 'Chinese', '简体中文', array(
-	'Class:lnkPersonToTeam' => '关联 个体/团队',
+	'Class:lnkPersonToTeam' => '关联个体/团队',
```
Comment (chileeb) [inline datamodels/2.x/itop-structure/dictionaries/zh_cn.dict.itop-structure.php]: 人员

File: datamodels/2.x/itop-structure/dictionaries/zh_cn.dict.itop-structure.php [hunk 12]
```diff
@@ -369,13 +369,13 @@
...
 	'Person:personal_info' => '个人信息',
 	'Person:notifiy' => '通知',
 ));
 
 // Themes
 Dict::Add('ZH CN', 'Chinese', '简体中文', array(
-	'theme:fullmoon' => 'Full moon~~',
-	'theme:test-red' => 'Test instance (Red)~~',
+	'theme:fullmoon' => '满月',
+	'theme:test-red' => '测例 (红色)',
```
Comment (chileeb) [inline datamodels/2.x/itop-structure/dictionaries/zh_cn.dict.itop-structure.php]: 测试（红色）

**Review by chileeb — CHANGES_REQUESTED** [review]

**Comment by Molkobain** [comment]
Thanks the PR! Let us know (via a mention) when you all agree and the PR is ready for integration :)

File: datamodels/2.x/combodo-db-tools/dictionaries/zh_cn.dict.combodo-db-tools.php (hunk 1 above)
Comment (bdejin) [reply datamodels/2.x/combodo-db-tools/dictionaries/zh_cn.dict.combodo-db-tools.php]: okay

File: datamodels/2.x/itop-config-mgmt/dictionaries/zh_cn.dict.itop-config-mgmt.php (hunk 2 above)
Comment (bdejin) [reply datamodels/2.x/itop-config-mgmt/dictionaries/zh_cn.dict.itop-config-mgmt.php]: okay

File: datamodels/2.x/itop-incident-mgmt-itil/dictionaries/zh_cn.dict.itop-incident-mgmt-itil.php (hunk 4 above)
Comment (bdejin) [reply datamodels/2.x/itop-incident-mgmt-itil/dictionaries/zh_cn.dict.itop-incident-mgmt-itil.php]: 创建于聊天工具沟通后的事件

File: datamodels/2.x/itop-incident-mgmt-itil/dictionaries/zh_cn.dict.itop-incident-mgmt-itil.php (hunk 3 above)
Comment (bdejin) [reply datamodels/2.x/itop-incident-mgmt-itil/dictionaries/zh_cn.dict.itop-incident-mgmt-itil.php]: okay

File: datamodels/2.x/itop-portal-base/dictionaries/zh_cn.dict.itop-portal-base.php (hunk 5 above)
Comment (bdejin) [reply datamodels/2.x/itop-portal-base/dictionaries/zh_cn.dict.itop-portal-base.php]: okay

File: datamodels/2.x/itop-structure/dictionaries/zh_cn.dict.itop-structure.php (hunk 11 above)
Comment (bdejin) [reply datamodels/2.x/itop-structure/dictionaries/zh_cn.dict.itop-structure.php]: okay

File: datamodels/2.x/itop-structure/dictionaries/zh_cn.dict.itop-structure.php (hunk 10 above)
Comment (bdejin) [reply datamodels/2.x/itop-structure/dictionaries/zh_cn.dict.itop-structure.php]: 绝大部分按照你的建议做了修正，同时修改了其他地方相同的翻译。我觉得之后可以全部检查下，保证所有类似的地方翻译一致。我们可以做个“中文翻译指南” :-)

Comment (bdejin) [reply datamodels/2.x/itop-structure/dictionaries/zh_cn.dict.itop-structure.php]: 这次时间仓促，主要做了缺失翻译的补充，来赶上3.2发布。之后可以全部检查下现有翻译。

**Review by chileeb — APPROVED** [review]
good job!

**Comment by bdejin** [comment]
@Molkobain I think we're good to go. The review is done - thanks @chileeb for your time.

**Comment by Molkobain** [comment]
Great job guys, thanks! It should be processed next week :)

**Comment by bdejin** [comment]
> Great job guys, thanks! It should be processed next week :)

@Molkobain ooops, I did a sync for my fork and the changes were gone and PR closed. :-( How can I revert this......

**Comment by Molkobain** [comment]
> > Great job guys, thanks! It should be processed next week :)
> 
> @Molkobain ooops, I did a sync for my fork and the changes were gone and PR closed. :-( How can I revert this......

Seems like you managed to re-open it :)

**Comment by Molkobain** [comment]
Accepted during technical review

**Comment by bdejin** [comment]
> > > Great job guys, thanks! It should be processed next week :)
> > 
> > 
> > @Molkobain ooops, I did a sync for my fork and the changes were gone and PR closed. :-( How can I revert this......
> 
> Seems like you managed to re-open it :)

Yep. Had some dig and got it back to the right trail.

**Comment by Molkobain** [comment]
Accepted during functional review

**Pull request merged**